docs: add "Connect to ScyllaDB Cloud Serverless" (#211)
scylladb/python-driver: docs/scylla_cloud_serverless.rst

ScyllaDB Cloud Serverless
-------------------------

> On ScyllaDB Cloud you can launch `serverless databases <https://cloud.docs.scylladb.com/stable/serverless/index.html>`_. 

Reviewer: ```suggestion
With ScyllaDB Cloud, you can deploy `serverless databases <https://cloud.docs.scylladb.com/stable/serverless/index.html>`_. 
```

> Python Driver supports connecting to those databases by configuring the driver to use the downloaded connection bundle.

Reviewer: ```suggestion
The Python driver allows you to connect to a serverless database by utilizing the connection bundle you can download via the **Connect>Python** tab in the Cloud application.
```

Here’s a short program that connects to a ScyllaDB Cloud serverless database and prints metadata about the cluster:

.. code-block:: python

    from cassandra.cluster import Cluster, ExecutionProfile, EXEC_PROFILE_DEFAULT
    from cassandra.policies import DCAwareRoundRobinPolicy, TokenAwarePolicy

    PATH_TO_BUNDLE_YAML = '/file/downloaded/from/cloud/connect-bundle.yaml'

    def get_cluster():
        profile = ExecutionProfile(
            load_balancing_policy=TokenAwarePolicy(
                DCAwareRoundRobinPolicy(local_dc='us-east-1')
            )
        )

        return Cluster(
            execution_profiles={EXEC_PROFILE_DEFAULT: profile},
            scylla_cloud=PATH_TO_BUNDLE_YAML,
        )

    print('Connecting to cluster')
    cluster = get_cluster()
    session = cluster.connect()

    print('Connected to cluster', cluster.metadata.cluster_name)

    print('Getting metadata')
    for host in cluster.metadata.all_hosts():
        print('Datacenter: {}; Host: {}; Rack: {}'.format(
            host.datacenter, host.address, host.rack)
        )

    cluster.shutdown()

> By providing a ``scylla_cloud`` parameter to the :class:`~.Cluster` constructor,

Reviewer: ```suggestion
By providing the ``scylla_cloud`` parameter to the :class:`~.Cluster` constructor,
```
